build_engine_test_images/terraform/aws/ubuntu/main.tf

```diff
@@ -0,0 +1,221 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.aws_region
+  access_key = var.build_engine_aws_access_key
+  secret_key = var.build_engine_aws_secret_key
+}
+
+# Create a random string suffix for instance names
+resource "random_string" "random_suffix" {
+  length           = 8
+  special          = false
+  lower            = true
+  upper            = false
+}
+
+# Create a VPC
+resource "aws_vpc" "build_engine_aws_vpc" {
+  cidr_block = "10.0.0.0/16"
+
+  tags = {
+    Name = "${var.build_engine_aws_vpc_name}-${random_string.random_suffix.id}"
+  }
+}
+
+# Create internet gateway
+resource "aws_internet_gateway" "build_engine_aws_igw" {
+  vpc_id = aws_vpc.build_engine_aws_vpc.id
+
+  tags = {
+    Name = "build_engine_igw"
+  }
+}
+
+# Create controlplane security group
```

Comment:
There are some "controlplane" appeared in this file but we're not creating a control plane.

Reply:
Renamed controlplane to build_node